### ISSUE TITLE
Upload should inline the Wasm

### ISSUE THREAD

**Issue by deuszx** [Cardinal-Cryptography/ink-wrapper#70]
When specifying the `--wasm-path` of the contract, the new method `fn upload` will be generated. It will use `include_bytes!(wasm_path)` which includes bytes of the contract. It's very fragile since it requires that at the time of calling the `ink-wrapper`, the `--wasm-path /path/to/wasm/file` is correct with regards to where it will be used, not where it's currently called from. I.e. the relative path to the Wasm file MUST be specified for the call time, not generation time. See https://github.com/Cardinal-Cryptography/ink-wrapper/blob/aleph-client-compat/Makefile#L38-L50 for example how this path changes.

Instead, we could simply inline the whole body of the contract as an array in the same file as the wrappers. Or, alternatively, inline it in a separate file but output it alongside the wrapper and call `include_bytes!` on it.

**Comment by obrok** [comment]
Yeah, I went back-and-forth on this initially. Didn't want to blow up the generate file too much with the full WASM, but it has the downside as you describe. I think if we generate a crate this would be neatly resolved.
